contents/teams/words-pictures/index.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: Words & Pictures
+template: team
+---
+
+## Values
+
+### 1. Progress is always more important than process
+
+We can be a chaotic team, but that's OK because we're also good at getting stuff done. We'll create process when it's helpful, but if it's a choice between doing something good/fast and doing something "properly", we'll choose good/fast. We encourage each other not to ask for permission on things. 
+
+### 2. Sparking joy is a multidisciplinary, inclusive job
+
+We're called Words & Pictures, but that's not all we do. We think in animations, emails, events, songs, videos, vibes, and more. There's no such thing as "just a writer" or "just a designer", and everyone's feedback is valued equally on all projects.
```

Comment:
'We think in....' etc. is a bit twee for my liking. I think this point would be stronger if it's more around continuing to push the brand further - let's keep being much more fun, weird and joyful than every other generic software company out there.